frontend/public/components/environment.jsx

```diff
@@ -44,212 +45,214 @@ export const envVarsToArray = (initialPairObjects) => {
   return [getPairsFromObject(initialPairObjects)];
 };
 
-export class EnvironmentPage extends PromiseComponent {
+const stateToProps = ({k8s}, {obj}) => ({
+  model: k8s.getIn(['RESOURCES', 'models', referenceFor(obj)]) || k8s.getIn(['RESOURCES', 'models', obj.kind]),
```

Comment:
We should really modify `connectToModel` so that it can be used with `obj: K8sResourceKind` as well so we don't have to duplicate this logic everywhere. I'll look into it.

Reply:
@alecmerdler thanks! Also, I fixed the build error with a type annotation in the jsx env component.